### PR TITLE
add Andreea D

### PR DESCRIPTION
**Pull request by eigidi**
Merci !

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -14312,6 +14312,83 @@ var data = {
...
+        {
+          title: ' L’architecture d’un chatbot',
+          abstract:
+            "Le chatbot est une application qui permet d'interagir avec l'utilisateur à travers une interface conversationnelle, qu'elle soit textuelle ou vocale. Son 'intelligence' réside dans la compréhension du langage humain, l'interprétation des connaissances et l'apprentissage dans le temps, tous les chatbots n'étant pas taillés pareil. Quelles sont les architectures des chatbots et quels sont les outils et les compétences dont on a besoin pour les réaliser ?",
+          tags: ['Intelligence artificielle', 'Chatbot', 'Architecture', 'NLP'],
+          lang: ['fr', 'en'],
+        },
+      ],
+      cities: [
+        ('Paris': {
```
Comment (dadoonet) [inline baggers.js]: Could you fix that? It should be just "Paris". See the other speakers.

**Review by dadoonet — CHANGES_REQUESTED** [review]

File: baggers.js (hunk 1 above)
Comment (eigidi) [reply baggers.js]: done, thx

**Comment by dadoonet** [comment]
Any idea why the other speakers have been touched? (Reformat).
Could you rollback that and just add your own change?

**Review by dadoonet — CHANGES_REQUESTED** [review]
Please rollback the changes to other speakers that this last commit added.

**Review by dadoonet — APPROVED** [review]
Welcome!

**Pull request merged**